CIP-1694/README.fr.md

```diff
@@ -0,0 +1,1185 @@
+---
+CIP: 1694
+Titre: Un premier pas vers une gouvernance décentralisée on-chain
```

Comment:
```suggestion
Title: Un premier pas vers une gouvernance décentralisée on-chain
```
Agreement above was that keys would be in English.  Both languages look natural for some of these keys (and `Revision` is the same in both languages) so this was an easy one to miss. 🤓

Reply:
oupsy